specification/operationalinsights/resource-manager/readme.go.md

```diff
@@ -27,6 +27,7 @@ batch:
   - tag: package-2021-12-01-preview
   - tag: package-2020-10
   - tag: package-2020-08
+  - tag: package-2022-10
```

Comment:
could we revert the changes in this file? These changes serve the track 1 Go SDK which is going to be deprecated very soon and we no longer release new versions for it.

Reply:
Done.